msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -1,170 +1,180 @@
 ---
-title: Import External Platform Messages
-description: Learn how to use Microsoft Graph to import messages such as message history and data from an third-party platform to Teams.
+title: Import external platform messages to Teams with Microsoft Graphs
+description: Learn how to use Microsoft Graph to import messages such as message history and data from any third-party platform to Teams.
 ms.localizationpriority: high
-author: "akjo"
+author: "surbhigupta"
 ms.topic: overview
-ms.owner: vishachadha
-ms.date: 09/02/2022
+ms.owner: mehakagarwal
+ms.date: 11/10/2025
 ---
 
-# Import third-party platform messages to Teams using Microsoft Graph
+# Import external messages into Teams with Microsoft Graph APIs
 
-With Microsoft Graph, you can migrate users' existing message history and data from an external system into a Teams channel. By enabling the recreation of a third-party platform messaging hierarchy inside Teams, users can continue their communications in a seamless manner and proceed without interruption.
+Use Microsoft Graph to import users' existing message history and data from any third-party platform into Teams. Users can continue their conversations seamlessly without interruption, as their messaging hierarchy from the third-party platform is recreated directly within Teams.
 
 > [!NOTE]
-> In the future, Microsoft may require you or your customers to pay additional fees based on the amount of data imported.
+> In the future, Microsoft might require you or your customers to pay extra fees based on the amount of data imported.
 
-## Import overview
+## Permissions
 
-At a high level, the import process consists of the following:
+|ScopeName|DisplayName|Type|APIs covered|
+|---------|---------|---------|---------|
+| `Teamwork.Migrate.All`| Manage migration to Microsoft Teams | Application-only|`POST/teams`|
 
-1. [Create a team with a back-in-time timestamp](#step-1-create-a-team).
-1. [Create a channel with a back-in-time timestamp](#step-2-create-a-channel).
-1. [Import external back-in-time dated messages](#step-3-import-messages).
-1. [Complete the team and channel migration process](#step-4-complete-migration-mode).
-1. [Add team members](#step-five-add-team-members).
+> [!NOTE]
+> Delegated authentication isn't supported.
+>
+
+## Supported channel and chat types
+
+|Entities |Sub type  |Migration mode support |Notes|
+|---------|---------|---------|---------|
+|**Channels** | Standard, Private, Shared | New and existing | Channels must be created or already in migration mode |
```

Comment:
We should differentiate and define "new" and "existing" here.

Technically ImportV2 API allows any existing chat/channels to enter migration mode, whether its created 1 second ago or 5 years old.

While import V1 API allows users to create "new" team and standard channel to be created from scratch in migration mode.